app/graphql/schema/mutations/update-customer-query-ticket.js

```diff
@@ -0,0 +1,38 @@
+const cosmos = require('../../../cosmos')
+const { cosmosConfig } = require('../../../config')
+const { customerQueryByTicketId } = require('../queries')
+
+const updateCustomerQueryTicket = async (_root, args, context) => {
+  const { queriesDatabase } = await cosmos()
+
+  const item = {
+    ticketId: args.ticketId,
+    internalUser: args.internalUser,
+    name: args.name,
+    heading: args.heading,
+    body: args.body
+  }
+
+  const response = await queriesDatabase
+    .container(cosmosConfig.queriesContainer)
+    .items.upsert(item)
```

Comment:
Is this updating the original item in the database? When testing i ended up with three unique items in the database. I expected there to be one item which has the responses appended to it. Is that the same behaviour you get?

Reply:
yeah, I get that too, I was getting the same before when I was using the `create` function instead of `upsert`.